Date: Nov 1, 2022 12:23 PM
From: Arjun <arjun_daasuramdass@comcast.com>
Subject: [PATCH] RDKALL-1221: Create separate CA bundles for browser apps
        as below removed cacert.pem as redundant heavy dust
        install browser based certs as per definition for browser
        based apps separately.
	This patch is made compatible to kirkstone Yocto version.
Source: COMCAST 02060561db73c28bcb9dff8d241c2275c6ad9b47 Mar 22, 2019 2:49 PM
Signed-off-by: Arjun <arjun_daasuramdass@comcast.com>
---

--- a/sbin/update-ca-certificates	2022-10-31 11:35:36.417666102 +0000
+++ b/sbin/update-ca-certificates	2022-10-31 11:38:35.772935860 +0000
@@ -26,6 +26,7 @@
 default=0
 CERTSCONF=$SYSROOT/etc/ca-certificates.conf
 CERTSDIR=$SYSROOT/usr/share/ca-certificates
+CERTSDIRBUNDLE=$SYSROOT/usr/share/ca-certificates/ca-certificates.crt
 LOCALCERTSDIR=$SYSROOT/usr/local/share/ca-certificates
 CERTBUNDLE=ca-certificates.crt
 ETCCERTSDIR=$SYSROOT/etc/ssl/certs
@@ -226,9 +227,10 @@
 if [ -f "$TEMPBUNDLE" ]
 then
   chmod 0644 "$TEMPBUNDLE"
-  mv -f "$TEMPBUNDLE" "$CERTBUNDLE"
+  mv -f "$TEMPBUNDLE" "$CERTSDIRBUNDLE"
+  ln -sf "${CERTSDIRBUNDLE##$SYSROOT}" "$CERTBUNDLE"
   # Restore proper SELinux label after moving the file
-  [ -x /sbin/restorecon ] && /sbin/restorecon "$CERTBUNDLE" >/dev/null 2>&1
+  [ -x /sbin/restorecon ] && /sbin/restorecon "$CERTSDIRBUNDLE" >/dev/null 2>&1
 fi
 
 echo "$ADDED_CNT added, $REMOVED_CNT removed; done."
